assets/js/xrp-ledger-toml-checker.js

```diff
@@ -0,0 +1,214 @@
+const CLASS_GOOD = "badge badge-success"
+const CLASS_BAD = "badge badge-danger"
+
+const ACCOUNT_FIELDS = [
+  "address",
+  "network",
+  "desc"
+]
+const VALIDATOR_FIELDS = [
+  "public_key",
+  "network",
+  "owner_country",
+  "server_country",
+  "unl"
+]
+const PRINCIPAL_FIELDS = [
+  "name",
+  "email"
+]
+const SERVER_FIELDS = [
+  "json_rpc",
+  "ws",
+  "peer",
+  "network"
+]
+const CURRENCY_FIELDS = [
+  "code",
+  "display_decimals",
+  "issuer",
+  "network",
+  "symbol"
+]
+
+function VerificationError(message, tips) {
+  this.name = "VerificationError"
+  this.message = message || ""
+  this.tips = tips || ""
+}
+VerificationError.prototype = Error.prototype
+
+function makeLogEntry(text, raw) {
+  let log
+  if (raw) {
+    log = $('<li></li>').appendTo('#log').append(text)
+  } else {
+    log = $('<li></li>').text(text+" ").appendTo('#log')
+  }
+  log.resolve = function(text) {
+    return $('<span></span>').html(text).appendTo(log)
+  }
+  return log
+}
+
+function fetch_file() {
+  const domain = $('#domain').val()
+  const url = "https://" + domain + TOML_PATH
+
+  const log = makeLogEntry('Checking ' + url + '...')
+  $.ajax({
+    url: url,
+    dataType: 'text',
+    success: function(data) {
+      log.resolve('FOUND').addClass(CLASS_GOOD)
+      parse_xrpl_toml(data, domain)
+    },
+    error: function(jqxhr, status, error) {
+      switch (status) {
+        case 'timeout':
+          err = 'TIMEOUT'
+          break
+        case 'abort':
+          err = 'ABORTED'
+          break
+        case 'error':
+          err = 'ERROR'
+          break
+        default:
+          err = 'UNKNOWN'
+      }
+      log.resolve(err).addClass(CLASS_BAD).after(TIPS)
+    }
+  })
+}
+
+async function parse_xrpl_toml(data, domain) {
+  let parsed
+  let log1 = makeLogEntry("Parsing TOML data...")
+  try {
+    parsed = TOML(data)
+    log1.resolve("SUCCESS").addClass(CLASS_GOOD)
+  } catch(e) {
+    log1.resolve(e).addClass(CLASS_BAD)
+    return
+  }
+
+  console.log(parsed)
+
+  if (parsed.METADATA.modified) {
+    const mod_log = makeLogEntry("Modified date: ")
+    try {
+      mod_log.resolve(parsed.METADATA.modified.toISOString()).addClass(CLASS_GOOD)
+    } catch(e) {
+      mod_log.resolve("INVALID").addClass(CLASS_BAD)
+    }
+  }
+
+  async function list_entries(name, list, fields, validate) {
+    let list_wrap = $("<p>"+name+"</p>")
+    let list_ol = $("<ol>").appendTo(list_wrap)
+    for (i=0; i<list.length; i++) {
+      let entry_wrap = $("<li>").appendTo(list_ol)
+      let entry_def = $("<ul class='mb-3'>").appendTo(entry_wrap)
+      let entry = list[i]
+      for (j=0; j<fields.length; j++) {
+        let fieldname = fields[j]
+        if (entry[fieldname] !== undefined) {
+          let field_def = $("<li><strong>"+fieldname+":</strong> ").appendTo(entry_def)
```

Comment:
In `":</strong> "`, the space is ignored/removed by jQuery. Simple fix: Move it to before `</strong>` like so:

`": </strong>"`